apps/nextjs-website/src/lib/fetch/types/mui/FeatureItemIcon.ts

```diff
@@ -0,0 +1,12 @@
+import * as t from 'io-ts';
+
+export const FeatureItemMUIIconCodec = t.keyof({
+  HubOutlined: null,
+  AutoFixHighOutlined: null,
+  MarkunreadMailboxOutlined: null,
+  CheckCircleOutlined: null,
+  SavingsOutlined: null,
+  HourglassEmptyRounded: null,
+  EnergySavingsLeafOutlined: null,
+  CloudOutlined: null,
+});
```

Comment:
It seems like the icons are specific to the FeatureItem element, so it would be best to keep them close to the component and not export them.